glusterd2/commands/volumes/volume-reset.go

```diff
@@ -89,17 +90,32 @@ func volumeResetHandler(w http.ResponseWriter, r *http.Request) {
 		restutils.SendHTTPError(ctx, w, http.StatusInternalServerError, err)
 		return
 	}
-
+	//save volume information for transaction failure scenario
+	if err := txn.Ctx.Set("oldvolinfo", oldvolinfo); err != nil {
+		logger.WithError(err).Error("failed to set oldvolinfo in transaction context")
```

Comment:
You can remove this log. Or..at least maintain uniformity with `txn.Ctx.Set` done on line 115 below.

Reply:
done